### PR TITLE
Add warning message to add allele specific annotation group

### PR DESCRIPTION
**Pull request by ronlevine**
Implements #1037.

### CODE REVIEW

**Comment by ronlevine** [comment]
@lbergelson Please review.

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java [hunk 1]
```diff
@@ -210,4 +209,15 @@ public void testIndexIsCreated(){
         runCommandLine(args);
         Assert.assertTrue(index.exists());
     }
+
+    @Test
+    public void testMissingAlleleSpecificAnnotationGroup() throws IOException {
```
Comment (lbergelson) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java]: This test doesn't assert anything?

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/ReferenceConfidenceVariantContextMerger.java [hunk 2]
```diff
@@ -368,8 +372,9 @@ private static void removeStaleAttributesAfterMerge(final Map<String, Object> at
                     values.add(Integer.parseInt(stringValue));
                 }
             } catch (final NumberFormatException e) {
-                // nothing to do
-                //TODO This comes directly from gatk3, we should determine if we should log this or if it happens so much it is not useful to log
+                final String baseMsg = "Remaining (non-reducible) annotations are assumed to be ints or doubles, but " + value + " doesn't parse and will not be annotated in the final VC.";
+                final String msg = value.toString().contains("|") ? baseMsg + " Add -G Standard -G AS_Standard to the command to annotate in the final VC." : baseMsg;
```
Comment (droazen) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/ReferenceConfidenceVariantContextMerger.java]: This seems like a pretty brittle way of detecting whether AS annotations are present...

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java (hunk 1 above)
Comment (lbergelson) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java]: You can use BaseTest.captureStdout.

**Comment by droazen** [comment]
@ronlevine Migration instructions for this branch: https://github.com/broadinstitute/gatk/wiki/Migrating-branches-from-gatk-protected-to-gatk

**Comment by ronlevine** [comment]
The branch has been migrated to the `gatk` repository.

**Comment by ronlevine** [comment]
Closing since migrated to https://github.com/broadinstitute/gatk/pull/3042.

**Pull request closed without merging**